### PR TITLE
Demo flag

### PR DESCRIPTION
**Pull request by muselesscreator**
Sets the frontend to load the "is this the demo version" flag from initialize api call. (not sure where this ticket went)

Also updates the on-close behavior of the review modal to check if there are ANY active grades to display the modal, and dispatches a new (mocked) batchUnlock api method.
[AU-578](https://openedx.atlassian.net/browse/AU-578

Also fixed the image assets URL.
[AU-575](https://openedx.atlassian.net/browse/AU-575)

### CODE REVIEW

File: src/containers/ListView/__snapshots__/EmptySubmission.test.jsx.snap [hunk 1]
```diff
@@ -6,7 +6,7 @@ exports[`EmptySubmission component component snapshot 1`] = `
 >
   <img
     alt="empty state"
-    src="assets/empty-state.svg"
+    src="icon/mock/path"
```
Comment (leangseu-edx) [inline src/containers/ListView/__snapshots__/EmptySubmission.test.jsx.snap]: where is this coming from?

File: src/data/redux/thunkActions/app.test.js [hunk 2]
```diff
@@ -23,15 +23,17 @@ describe('app thunkActions', () => {
       expect(typeof dispatchedAction.initializeApp.onSuccess).toEqual('function');
     });
     describe('on success', () => {
-      test('loads oraMetadata, courseMetadata and list data', () => {
+      test('loads isEnabled, oraMetadata, courseMetadata and list data', () => {
```
Comment (leangseu-edx) [inline src/data/redux/thunkActions/app.test.js:26]: 👍

File: src/data/redux/thunkActions/requests.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ export const fetchSubmission = ({ submissionUUID, ...rest }) => (dispatch) => {
 };
 
 /**
- * Tracked initializeApp api method.  tracked to the `initialize` request key.
+ * Tracked setLock api method.  tracked to the `setLock` request key.
```
Comment (leangseu-edx) [inline src/data/redux/thunkActions/requests.js:81]: 🏆

File: src/data/services/lms/api.js [hunk 4]
```diff
@@ -92,6 +93,15 @@ const unlockSubmission = (submissionUUID) => client().delete(
   }),
 ).then(response => response.data);
 
+/**
+ * batchUnlockSubmissions(submissionUUIDs)
+ * @param {string[]} submissionUUIDs - list of submission uuids
+ */
+const batchUnlockSubmissions = (submissionUUIDs) => {
+  console.log({ batchUnlockSubmissions: submissionUUIDs });
```
Comment (leangseu-edx) [inline src/data/services/lms/api.js:101]: to be remove

File: src/utils/keyStore.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import StrictDict from './StrictDict';
+
+const keyStore = (collection) => StrictDict(
+  Object.keys(collection).reduce(
+    (obj, key) => ({ ...obj, [key]: key }),
+    {},
+  ),
+);
+
+export default keyStore;
```
Comment (leangseu-edx) [inline src/utils/keyStore.js:10]: This file isn't being use.

**Review by leangseu-edx — CHANGES_REQUESTED** [review]
Look good. Just remove the debris.

File: src/data/services/lms/api.js (hunk 4 above)
Comment (muselesscreator) [reply src/data/services/lms/api.js:101]: leaving this until we have the actual API hook connected as the interim way of verifying the action worked.

File: src/containers/ListView/__snapshots__/EmptySubmission.test.jsx.snap (hunk 1 above)
Comment (muselesscreator) [reply src/containers/ListView/__snapshots__/EmptySubmission.test.jsx.snap]: not sure where specifically, but I should update with intentional mock.

File: src/utils/keyStore.js (hunk 5 above)
Comment (muselesscreator) [reply src/utils/keyStore.js:10]: This was introduced in the TnL stuff and I want to start using it, especially for places where we are spying on things

Comment (muselesscreator) [reply src/utils/keyStore.js:10]: added because I intend to use soon

**Pull request merged**